lawnchair/src/app/lawnchair/ui/preferences/AppDrawerPreferences.kt

```diff
@@ -61,18 +55,34 @@ fun AppDrawerPreferences() {
             )
             SuggestionsPreference()
         }
+
+        PreferenceGroup(heading = stringResource(id = R.string.hidden_apps_label)) {
+            val hiddenApps = prefs2.hiddenApps.getAdapter().state.value
+            val showHiddenAppsInSearch = prefs2.showHiddenAppsInSearch.getAdapter()
+            val enableSmartHide = prefs2.enableSmartHide.getAdapter()
+            NavigationActionPreference(
+                label = stringResource(id = R.string.hidden_apps_label),
+                subtitle = resources.getQuantityString(R.plurals.apps_count, hiddenApps.size, hiddenApps.size),
+                destination = subRoute(name = AppDrawerRoutes.HIDDEN_APPS),
+            )
+            SwitchPreference(
+                label = stringResource(id = R.string.show_hidden_apps_search),
+                adapter = showHiddenAppsInSearch,
+            )
+            SwitchPreference(
+                label = stringResource(id = R.string.show_enable_smart_hide),
+                adapter = enableSmartHide,
+                enabled = !showHiddenAppsInSearch.state.value
+            )
```

Comment:
```suggestion
            if (hiddenApps.size > 0) {
                SwitchPreference(
                    label = stringResource(id = R.string.show_hidden_apps_search),
                    adapter = showHiddenAppsInSearch,
                )
                SwitchPreference(
                    label = stringResource(id = R.string.show_enable_smart_hide),
                    adapter = enableSmartHide,
                    enabled = !showHiddenAppsInSearch.state.value
                )
            }
```

(Not sure if we should wrap this in an `ExpandAndShrink {}` though)

Reply:
Thanks for the suggestion , changes applied.